packages/x-components/src/x-modules/next-queries/components/next-query-preview.vue

```diff
@@ -209,4 +224,49 @@ In this example, the ID of the results will be rendered along with the name.
   };
 </script>
 ```
+
+### Play with props
+
+In this example, the suggestions has been limited to render a maximum of 4 items.
+
+```vue
+<template>
+  <NextQueryPreview
+    :maxItemsToRender="maxItemsToRender"
+    :suggestion="suggestion"
+    #result="{ result }"
+  >
+    <BaseGrid #default="{ item }">
```

Comment:
How is getting this `BaseGrid` the items to show? Shouldn't we pass `NextQueryPreview`'s `results` from the default slot?